### PR TITLE
Fixed edition-header h3 spacing

### PR DESCRIPTION
**Pull request by Yashs911**
Related to #3713

### Stakeholders

@jdlrobson

### CODE REVIEW

File: static/css/components/work.less [hunk 1]
```diff
@@ -29,7 +29,7 @@
 // Center (editionAbout)
 div.editionAbout {
```
Comment (jdlrobson) [inline static/css/components/work.less:30]: I think using h3.edition-header should be more than enough here.

I would recommend dropping div.editionAbout from the selector. We are trying to move towards BEM and simplify these selectors.

Comment (Yashs911) [reply static/css/components/work.less:30]: @jdlrobson will this work
image

Comment (Yashs911) [reply static/css/components/work.less]: or you want me to remove the div.editionAbout h3 selector?

Comment (Yashs911) [reply static/css/components/work.less:30]: Done.

File: openlibrary/templates/type/edition/view.html [hunk 2]
```diff
@@ -412,7 +412,7 @@ <h3 class="collapse">$_("Contributors")</h3>
 
           $if has_any("physical_format", "pagination", "physical_dimensions", "weight"):
             <div class="section">
-                <h3 class="list-header collapse">$_("The Physical Object")</h3>
+                <h3 class="edition-header">$_("The Physical Object")</h3>
```
Comment (jdlrobson) [inline openlibrary/templates/type/edition/view.html:415]: These should still be list-header.

The only issue is with the generic h3's that don't match list-header.

File: openlibrary/templates/type/edition/view.html [hunk 3]
```diff
@@ -424,7 +424,7 @@ <h3 class="list-header collapse">$_("The Physical Object")</h3>
 
 
           <div class="section">
-            <h3 class="list-header collapse">$_("ID Numbers")</h3>
+            <h3 class="edition-header">$_("ID Numbers")</h3>
```
Comment (jdlrobson) [inline openlibrary/templates/type/edition/view.html:427]: These should still be list-header.

The only issue is with the generic h3's that don't match list-header.

Comment (Yashs911) [reply openlibrary/templates/type/edition/view.html:427]: list-header or previous list-header collapse?

**Comment by Yashs911** [comment]
@jdlrobson I have made the required changes

File: openlibrary/templates/type/edition/view.html (hunk 3 above)
Comment (jdlrobson) [reply openlibrary/templates/type/edition/view.html]: The html for these is fine so retain as is.

File: openlibrary/templates/type/edition/view.html [hunk 4]
```diff
@@ -339,13 +339,13 @@ <h3>
           <hr>  
           $if edition.first_sentence:
             <div>
-            <h3 class="collapse">$_("First Sentence")</h3>
```
Comment (jdlrobson) [inline openlibrary/templates/type/edition/view.html:342]: `collapse` class is only styled on the homepage static/css/components/home.less
Has no impact on JS. Likely an old artifact from another time.

**Review by jdlrobson — APPROVED** [review]

**Pull request merged**